driver-core/src/test/functional/com/mongodb/ClusterFixture.java

```diff
@@ -102,6 +102,7 @@
     public static final String MONGODB_URI_SYSTEM_PROPERTY_NAME = "org.mongodb.test.uri";
     public static final String MONGODB_API_VERSION = "org.mongodb.test.api.version";
     public static final String MONGODB_TRANSACTION_URI_SYSTEM_PROPERTY_NAME = "org.mongodb.test.transaction.uri";
+    public static final String SERVERLESS_TEST_SYSTEM_PROPERTY_NAME = "org.mongodb.test.serverless";
```

Comment:
Introduced this property since there is no way to figure out from the URI that it's serverless instance